scenarios/kubernetes_verify.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Need to figure out why this only fails on travis
+# pylint: disable=bad-continuation
+
+"""Runs verify/test-go checks for kubernetes/kubernetes."""
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+
+
+BRANCH_VERSION = {
+    '1.2': '1.4',
+    '1.3': '1.4',
+    'master': '1.6',
+}
+
+VERSION_TAG = {
+    '1.4': '1.4-v20161130-8958f82',
+    '1.5': '1.5-v20161205-d664d14',
+    '1.6': '1.6-v20161205-ad918bc',
+}
+
+
+def check(*cmd):
+    """Log and run the command, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    subprocess.check_call(cmd)
+
+
+def main(branch, script, force):
+    """Test branch using script, optionally forcing verify checks."""
+    verify_branch = re.match(r'.*(master|\d+\.\d+)', branch)
+    if not verify_branch:
+        raise ValueError(branch)
+    tag = VERSION_TAG[BRANCH_VERSION.get(verify_branch.group(1))]
```

Comment:
assume default need to be set? 

i.e.`BRANCH_VERSION.get(verify_branch.group(1),verify_branch.group(1))`?